.teamcity/_self/projects/WPComTests.kt

```diff
@@ -208,174 +208,168 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
-                buildFinishedSuccessfully = true
-            }
-            commitStatusPublisher {
-                vcsRootExtId = "${Settings.WpCalypso.id}"
-                publisher = github {
-                    githubUrl = "https://api.github.com"
-                    authType = personalToken {
-                        token = "credentialsJSON:57e22787-e451-48ed-9fea-b9bf30775b36"
-                    }
-                }
-            }
-        }
-
-        failureConditions {
-            executionTimeoutMin = 20
-            // Don't fail if the runner exists with a non zero code. This allows a build to pass if the failed tests have
-            // been muted previously.
-            nonZeroExitCode = false
-
-            // Fail if the number of passing tests is 50% or less than the last build. This will catch the case where the test runner
-            // crashes and no tests are run.
-            failOnMetricChange {
-                metric = BuildFailureOnMetric.MetricType.PASSED_TEST_COUNT
-                threshold = 50
-                units = BuildFailureOnMetric.MetricUnit.PERCENTS
-                comparison = BuildFailureOnMetric.MetricComparison.LESS
-                compareTo = build {
-                    buildRule = lastSuccessful()
-                }
-            }
-        }
-
-        triggers {
-            schedule {
-                schedulingPolicy = daily {
-                    hour = 4
-                }
-                branchFilter = """
-                    +:trunk
-                """.trimIndent()
-                triggerBuild = always()
-                withPendingChangesOnly = false
-            }
-        }
-    }
+		artifactRules = """
+			reports => reports
+			logs.tgz => logs.tgz
+			screenshots => screenshots
+		""".trimIndent()
+
+		vcs {
+			root(Settings.WpCalypso)
+			cleanCheckout = true
+		}
+
+		params {
+			text(
+				name = "URL",
+				value = "https://wordpress.com",
+				label = "Test URL",
+				description = "URL to test against",
+				allowEmpty = false
+			)
+			checkbox(
+				name = "GUTENBERG_EDGE",
+				value = "false",
+				label = "Use gutenberg-edge",
+				description = "Use a blog with gutenberg-edge sticker",
+				checked = "true",
+				unchecked = "false"
+			)
+			checkbox(
+				name = "COBLOCKS_EDGE",
+				value = "false",
+				label = "Use coblocks-edge",
+				description = "Use a blog with coblocks-edge sticker",
+				checked = "true",
+				unchecked = "false"
+			)
+		}
+
+		steps {
+			bashNodeScript {
+				name = "Prepare environment"
+				scriptContent = """
+					export NODE_ENV="test"
+					export PLAYWRIGHT_BROWSERS_PATH=0
+
+					# Install modules
+					${_self.yarn_install_cmd}
+
+					# Build packages
+					yarn workspace @automattic/calypso-e2e build
+				"""
+			}
+			bashNodeScript {
+				name = "Run e2e tests ($viewportName)"
+				scriptContent = """
+					shopt -s globstar
+					set -x
+
+					cd test/e2e
+					mkdir temp
+
+					export LIVEBRANCHES=true
```

Comment:
I think this should be disabled. In Selenium+Magellan land this instructed the system to use live branches (aka calypso.live). (Although I don't know if this is actually used anymore)